source/management_console/index.js

```diff
@@ -38,6 +64,68 @@ proxy.on('error', function(e) {
   console.log(e);
 });
 
+var encodeBase64 = (function () {
```

Comment:
You don't need this to encode base64 in 'nodejs'.
A simple statement `Buffer.from("string").toString('base64')` should work too.